atc/db/worker_factory.go

```diff
@@ -321,36 +321,17 @@ func (f *workerFactory) SaveWorker(atcWorker atc.Worker, ttl time.Duration) (Wor
 	return savedWorker, nil
 }
 
-func (f *workerFactory) FindWorkerForContainerByOwner(owner ContainerOwner, teamID int, tags atc.Tags) (Worker, bool, error) {
-	var teamWorkers int
-
-	err := psql.Select("COUNT (1)").
-		From("workers").
-		Where(sq.Eq{
-			"team_id": teamID,
-		}).
-		RunWith(f.conn).
-		Scan(&teamWorkers)
-	if err != nil {
-		return nil, false, err
-	}
-
+func (f *workerFactory) FindWorkersForContainerByOwner(owner ContainerOwner) ([]Worker, error) {
```

Comment:
ahh this makes a lot more sense now - no more team/tag filtering, just cross-referencing with compatible (/desireable) workers after finding the existing ones